src/settings/App.tsx

```diff
@@ -18,10 +18,11 @@ import AdminGroupSelect from './AdminGroupSelect'
 import SubAdminGroupSelect from './SubAdminGroupSelect'
 import { loadState } from '@nextcloud/initial-state'
 
+const bytesInOneGibibyte = Math.pow(1024, 3)
 const defaultQuotaOptions = {
-	'1 GB': 1073741274,
-	'5 GB': 5368709120,
-	'10 GB': 10737412742,
+	'1 GB': bytesInOneGibibyte,
+	'5 GB': bytesInOneGibibyte * 5,
+	'10 GB': bytesInOneGibibyte * 10,
```

Comment:
Should we change the unit for clarity?
```suggestion
	'1 GiB': bytesInOneGibibyte,
	'5 GiB': bytesInOneGibibyte * 5,
	'10 GiB': bytesInOneGibibyte * 10,
```

Reply:
I did that in the beginning, but then I figured out this is the style that is used everywhere.
I don't like it either as it is the worst of both worlds, but we'll just have to live with it.